docs/docs/feature/keymaps.md

```diff
@@ -46,11 +44,11 @@ in ZMK contains a set of bindings that bind a certain behavior to a certain key
 In addition to the base default layer (which can be changed), certain bound behaviors may also
 enable/disable additional layers "on top" of the default layer.
 
-When a key location is pressed/released, the stack of all active layers from "top to bottom" is used,
-and the event is sent to the behavior bound at that position in each layer, for it to perform whatever
-actions it wants to in reaction to the event. Those behaviors can choose to "handle" the event, and stop
-it from being passed to any lower layers, or may choose to "pass it along", and let the next layer
-in the stack _also_ get the event.
+When a key location is pressed or released, the stack of all active [layers](/docs/terms#layer) from "top to
+bottom" is used. The event is sent to the [behavior](/docs/terms#behavior) bound at that position in each
+layer. Those [behaviors](/docs/terms#behavior) can "handle" the event, stop it from being passed to any lower
+[layers](/docs/terms#layer), or choose to "pass it along" and let the [layer](/docs/terms#layer) below handle
+the event.
```

Comment:
```suggestion
layer. These [behaviors](/docs/terms#behavior) can "handle" the event, stopping it from being passed to any lower
[layers](/docs/terms#layer), or choose to "pass it along" and let the [layer](/docs/terms#layer) below handle
the event.
```

In its current form, I think it can be read as three options:

 - Handle
 - Stop
 - Pass

Rather than "Handle and stop" or "Pass". Swapping to "stopping" or some other reword here would help that.